modules/docs/lib/InformationHighlight/index.tsx

```diff
@@ -55,7 +54,7 @@ export const InformationHighlight = createContainer('section')({
   return (
     <Base
       as={Element}
-      cs={[containerStyles, containerModifiers({variant: model.state.variant})]}
+      {...informationHighlightStencil({variant: model.state.variant})}
```

Comment:
While a stencil does return a spreadable object, this pattern doesn't handle merging. There's a very simple way to break `InformationHighlight`:

```tsx
<InformationHighlight className="whoops" />
```

The `className` prop returned by the stencil will be overwritten by the incoming prop. At this point, `Base` isn't doing anything useful for us.

This is also why I'm not a fan of us using a `Base` component because even though we can use the `cs` prop, our component cannot handle the `cs` prop because it will get overridden instead  of merged. I think the safest way these components work is by using `handleCsProp` and dropping `Base`:


```tsx
return (
  <Element {...handleCsProp(props, informationHighlightStencil({variant: model.state.variant}))} />
```